src/twisted/trial/runner.py

```diff
@@ -126,20 +121,14 @@ def _importFromFile(fn, moduleName=None):
         moduleName = os.path.splitext(os.path.split(fn)[-1])[0]
     if moduleName in sys.modules:
         return sys.modules[moduleName]
-    if _PY35PLUS:
-        import importlib
-
-        spec = importlib.util.spec_from_file_location(moduleName, fn)
-        if not spec:
-            raise SyntaxError(fn)
-        module = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(module)
-        sys.modules[moduleName] = module
-    else:
-        import imp
-
-        with open(fn, 'r') as fd:
-            module = imp.load_source(moduleName, fn, fd)
+    import importlib
```

Comment:
these two importlib imports can move to the top (2)

Reply:
done